server/services/applicationService.ts

```diff
@@ -104,20 +104,69 @@ export default class ApplicationService {
   }
 
   private deleteOrphanedFollowOnAnswers(applicationData: AnyValue): AnyValue {
-    if (applicationData['funding-information']?.['funding-source']?.fundingSource === 'personalSavings') {
+    const deleteOrphanedFundingInformation = () => {
       delete applicationData['funding-information'].identification
       delete applicationData['funding-information']['alternative-identification']
     }
-    if (applicationData['equality-and-diversity-monitoring']?.['will-answer-equality-questions']?.willAnswer === 'no') {
+
+    const deleteOrphanedEqualityInformation = () => {
```

Comment:
could we move these function definitions to a utils file for greater readability?